core/common/src/test/java/alluxio/ConfigurationTest.java

```diff
@@ -11,319 +11,193 @@
-    Assert.assertNotNull(value);
-    Assert.assertEquals("org.apache.hadoop.fs.glusterfs.GlusterFileSystem", value);
-
-    boolean booleanValue = Configuration.getBoolean(Constants.ZOOKEEPER_ENABLED);
-    Assert.assertFalse(booleanValue);
-
-    booleanValue = Configuration.getBoolean(Constants.IN_TEST_MODE);
-    Assert.assertFalse(booleanValue);
-
-    int intValue = Configuration.getInt(Constants.NETWORK_HOST_RESOLUTION_TIMEOUT_MS);
-    Assert.assertEquals(Constants.DEFAULT_HOST_RESOLUTION_TIMEOUT_MS, intValue);
-
-    long longBytesValue =
-        Configuration.getBytes(Constants.USER_BLOCK_REMOTE_READ_BUFFER_SIZE_BYTES);
-    Assert.assertEquals(Constants.MB * 8, longBytesValue);
-
-    longBytesValue = Configuration.getBytes(Constants.NETWORK_THRIFT_FRAME_SIZE_BYTES_MAX);
-    Assert.assertEquals(Constants.MB * 16, longBytesValue);
-
-    int maxTry = Configuration.getInt(Constants.ZOOKEEPER_LEADER_INQUIRY_RETRY_COUNT);
-    Assert.assertEquals(10, maxTry);
   }
 
-  /**
-   * Tests the default properties for the master.
-   */
   @Test
-  public void masterDefaultTest() {
-    String alluxioHome = Configuration.get(Constants.HOME);
-    Assert.assertNotNull(alluxioHome);
-    Assert.assertEquals("/mnt/alluxio_default_home", alluxioHome);
-
-    String value = Configuration.get(Constants.MASTER_JOURNAL_FOLDER);
-    Assert.assertNotNull(value);
-    Assert.assertEquals(alluxioHome + "/journal/", value);
-
-    value = Configuration.get(Constants.MASTER_HOSTNAME);
-    Assert.assertNotNull(value);
-    Assert.assertEquals(NetworkAddressUtils.getLocalHostName(100), value);
-
-    value = Configuration.get(Constants.MASTER_FORMAT_FILE_PREFIX);
-    Assert.assertNotNull(value);
-    Assert.assertEquals(Constants.FORMAT_FILE_PREFIX, value);
-
-    value = Configuration.get(Constants.MASTER_ADDRESS);
-    Assert.assertNotNull(value);
-
-    value = Configuration.get(Constants.MASTER_BIND_HOST);
-    Assert.assertNotNull(value);
-    Assert.assertEquals(NetworkAddressUtils.WILDCARD_ADDRESS, value);
-
-    int intValue = Configuration.getInt(Constants.MASTER_RPC_PORT);
-    Assert.assertEquals(19998, intValue);
-
-    value = Configuration.get(Constants.MASTER_WEB_BIND_HOST);
-    Assert.assertNotNull(value);
-    Assert.assertEquals(NetworkAddressUtils.WILDCARD_ADDRESS, value);
-
-    intValue = Configuration.getInt(Constants.MASTER_WEB_PORT);
-    Assert.assertEquals(19999, intValue);
-
-    intValue = Configuration.getInt(Constants.WEB_THREAD_COUNT);
-    Assert.assertEquals(1, intValue);
-
-    intValue = Configuration.getInt(Constants.MASTER_HEARTBEAT_INTERVAL_MS);
-    Assert.assertEquals(Constants.SECOND_MS, intValue);
-
-    intValue = Configuration.getInt(Constants.MASTER_WORKER_THREADS_MIN);
-    Assert.assertEquals(512, intValue);
-
-    intValue = Configuration.getInt(Constants.MASTER_WORKER_TIMEOUT_MS);
-    Assert.assertEquals(300 * Constants.SECOND_MS, intValue);
+  public void getInt() {
```

Comment:
I think we might want to drop the naming convention since it seems superfluous when tests are already annotated with `@Test`.